### PR TITLE
Improve the quality of the documents

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

## Which problem is this PR solving?
- 

## Short description of the changes
- Perfect documentation.

### CODE REVIEW

**Comment by yuxiaobo96** [comment]
@objectiser 

I have a question.
Whether the folder "swagger-gen" and the folder "thrift-gen" do not need to be modified

File: swagger-gen/restapi/embedded_spec.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func init() {
         "description": "Returns a list of all service names associated with span endpoints.\n",
         "responses": {
           "200": {
-            "description": "Succes",
+            "description": "Success",
```
Comment (objectiser) [inline swagger-gen/restapi/embedded_spec.go]: This actual error comes from https://github.com/jaegertracing/jaeger-idl/blob/master/swagger/zipkin2-api.yaml#L21

File: swagger-gen/restapi/embedded_spec.go [hunk 2]
```diff
@@ -178,7 +178,7 @@ func init() {
           },
           {
             "type": "string",
-            "description": "Ex. ` + "`" + `http.uri=/foo and retried` + "`" + ` - If key/value (has an ` + "`" + `=` + "`" + `),\nconstrains against Span.tags entres. If just a word, constrains\nagainst Span.annotations[].value or Span.tags[].key. Any values are\nAND against eachother. This means a span in the trace must match\nall of these.\n",
+            "description": "Ex. ` + "`" + `http.uri=/foo and retried` + "`" + ` - If key/value (has an ` + "`" + `=` + "`" + `),\nconstrains against Span.tags entres. If just a word, constrains\nagainst Span.annotations[].value or Span.tags[].key. Any values are\nAND against each other. This means a span in the trace must match\nall of these.\n",
```
Comment (objectiser) [inline swagger-gen/restapi/embedded_spec.go]: https://github.com/jaegertracing/jaeger-idl/blob/master/swagger/zipkin2-api.yaml#L101

File: thrift-gen/zipkincore/ttypes.go [hunk 3]
```diff
@@ -730,7 +730,7 @@ func (p *BinaryAnnotation) String() string {
 // precise value possible. For example, gettimeofday or syncing nanoTime
 // against a tick of currentTimeMillis.
 //
-// For compatibilty with instrumentation that precede this field, collectors
+// For compatibility with instrumentation that precede this field, collectors
```
Comment (objectiser) [inline thrift-gen/zipkincore/ttypes.go]: https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/zipkincore.thrift#L305

File: thrift-gen/zipkincore/ttypes.go [hunk 4]
```diff
@@ -742,7 +742,7 @@ func (p *BinaryAnnotation) String() string {
 // precise measurement decoupled from problems of clocks, such as skew or NTP
 // updates causing time to move backwards.
 //
-// For compatibilty with instrumentation that precede this field, collectors
+// For compatibility with instrumentation that precede this field, collectors
```
Comment (objectiser) [inline thrift-gen/zipkincore/ttypes.go]: https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/zipkincore.thrift#L320

**Comment by yuxiaobo96** [comment]
Ok, thank you.

**Pull request merged**